### DISCUSSION TITLE
How can I write stream or buffer to client in route in appdir?

### DISCUSSION THREAD

**Discussion by taoqf** [vercel/next.js#48236] [Help] [answered]
### Summary

I cannot find a way to write buffer data to client through https://beta.nextjs.org/docs/api-reference/response
I want write buffer or stream to client. 
Is there a way or am I missing something?

**Comment by icyJoseph** [comment, marked as answer]
Do you mean this one? https://beta.nextjs.org/docs/routing/route-handlers#streaming

**Reply by taoqf** [reply]
Thanks, Yes, it should be.
